apis/vshn/v1/vshn_minio.go

```diff
@@ -247,3 +246,14 @@ func (v *VSHNMinio) GetPDBLabels() map[string]string {
 func (v *VSHNMinio) GetSecurity() *Security {
 	return &v.Spec.Parameters.Security
 }
+
+func (v *VSHNMinio) GetWorkloadPodTemplateLabelsManager() PodTemplateLabelsManager {
+	if v.GetInstances() == 1 {
+		return &DeploymentManager{}
+	}
+	return &StatefulSetManager{}
+}
```

Comment:
single instance minio is deployment and for multiple it's statefulset?

Reply:
yes